new-site/index.html

```diff
@@ -1,32 +1,162 @@
-<!doctype html>
-<html class="no-js" lang="en">
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Emily's page</title>
+  </head>
 
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
-  <meta name="description" content="">
+  <header class="header" id="header">
 
-  <meta property="og:title" content="">
-  <meta property="og:type" content="">
-  <meta property="og:url" content="">
-  <meta property="og:image" content="">
+    <div id="logo">
+      <header>
+        <h1>Frameworks</h1>
+    </header>
+    </div>
 
-  <link rel="icon" href="/favicon.ico" sizes="any">
-  <link rel="icon" href="/icon.svg" type="image/svg+xml">
-  <link rel="apple-touch-icon" href="icon.png">
+    <nav class="nav-bar">
+      <ul>
+        <li id="React">
+          <li class="dropdown">
+            <a href="https://reactjs.org/" class="dropbtn">React</a>
+            <div class="dropdown-content">
+              <a href="https://react.dev/learn" target="_blank">Learn</a>
+              <a href="https://react.dev/reference" target="_blank"
+              >Reference</a>
+              <a href="https://react.dev/community" target="_blank"
+              >Community</a>
+            </div>
+          </li>
+        </li>
 
-  <link rel="manifest" href="site.webmanifest">
-  <meta name="theme-color" content="#fafafa">
-</head>
+        <li><a href="https://vuejs.org/"  target="_self">Vue.js</a></li>
+        <li><a href="https://angular.io/"  target="_self">Angular</a></li>
 
-<body>
+      </ul>
+    </nav>
 
-  <!-- Add your site or application content here -->
-  <p>Hello world! This is HTML5 Boilerplate.</p>
-  <script src="js/app.js"></script>
+  </header>
 
-</body>
+  <body>
+    <div class="cards">
 
+
+    <div class="container" id="Angular" >
```

Comment:
Why do you need the `id` here? Also, perhaps the `container` class could just be called `card` because the thing is a card?